.rubocop.yml

```diff
@@ -4,4 +4,7 @@ AllCops:
 Metrics/LineLength:
   Max: 99
 Documentation:
-  Enabled: false
\ No newline at end of file
+  Enabled: false
```

Comment:
if this is an open source project, better force to leave a couple of comments to class.